model/models.urdf.xacro

```diff
@@ -66,6 +66,51 @@
 		<origin xyz="0 0 0.0145" rpy="0 0 0"/>
 	</xacro:property>
 
+<!-- http://www.ati-ia.com/products/ft/ft_models.aspx?id=Gamma -->
+<xacro:property name="gamma">
+			<inertial>
+				<mass value="0.255"/>
+				<origin xyz="0 0 0.01665"  />
+				<xacro:cylinder_inertia_def radius="0.0377" length="0.0333" mass="0.255"/>
+			</inertial>
+			<visual>
+				<!-- origin xyz="0 0 0.00725" rpy="0 0 0"/ -->
+				<origin xyz="0 0 0" rpy="0 0 0"/>
+				<geometry>
+					<!-- cylinder radius="0.0085" length="0.0145"/ -->
+					<mesh filename="package://force_torque_sensor/model/meshes/nano17.stl" scale="4.4 4.4 1"/>
+				</geometry>
+			</visual>
+			<collision>
+				<origin xyz="0 0 0.01665" rpy="0 0 0"/>
+				<geometry>
+					<cylinder radius="0.0377" length="0.005"/>
+				</geometry>
+			</collision>
+	</xacro:property>
+	<xacro:property name="gamma_platform">
+			<inertial>
+				<mass value="0.002"/>
+				<origin xyz="0 0 0.0025"  />
+				<xacro:cylinder_inertia_def radius="0.0377" length="0.005 " mass="0.255"/>
+			</inertial>
+			<visual>
+				<!-- origin xyz="0 0 0.00725" rpy="0 0 0"/ -->
+				<origin xyz="0 0 0.0025" rpy="0 0 0"/>
+				<geometry>
+					<cylinder radius="0.0377" length="0.005"/>
+				</geometry>
+			</visual>
+			<collision>
+				<origin xyz="0 0 0.0025" rpy="0 0 0"/>
+				<geometry>
+					<cylinder radius="0.0377" length="0.005"/>
+				</geometry>
+			</collision>
+	</xacro:property>
+	<xacro:property name="gamma_measure_frame">
+		<origin xyz="0 0 0.0145" rpy="0 0 0"/>
```

Comment:
According to the [drawing](http://www.ati-ia.com/app_content/Documents/9230-05-1329.auto.pdf), the sensing frame is at 33.3mm, not 14.5mm. I'm sending you the stl via email.